features/main/embedded.feature

```diff
@@ -0,0 +1,280 @@
+    And the header "Content-Type" should be equal to "application/ld+json; charset=utf-8"
+    And the JSON should be equal to:
+    """
+    {
+      "@context": "/contexts/RelatedDummy",
+      "@id": "/related_dummies/1",
+      "@type": "https://schema.org/Product",
+      "id": 1,
+      "name": "Hello",
+      "symfony": "symfony",
+      "dummyDate": null,
+      "thirdLevel": "/third_levels/1",
+      "relatedToDummyFriend": [],
+      "dummyBoolean": null,
+      "age": null
+    }
+    """
+
+  Scenario: Create an unnamed related dummy
+    When I add "Content-Type" header equal to "application/ld+json"
+    And I send a "POST" request to "/related_dummies" with body:
+    """
+    {"thirdLevel": "/third_levels/1"}
+    """
+    Then the response status code should be 201
+    And the response should be in JSON
+    And the header "Content-Type" should be equal to "application/ld+json; charset=utf-8"
+    And the JSON should be equal to:
+    """
+    {
+      "@context": "/contexts/RelatedDummy",
+      "@id": "/related_dummies/2",
+      "@type": "https://schema.org/Product",
+      "id": 2,
+      "name": null,
+      "symfony": "symfony",
+      "dummyDate": null,
+      "thirdLevel": "/third_levels/1",
+      "relatedToDummyFriend": [],
+      "dummyBoolean": null,
+      "age": null
+    }
+    """
+
+  Scenario: Create a dummy with relations
+    When I add "Content-Type" header equal to "application/ld+json"
+    And I send a "POST" request to "/dummies" with body:
+    """
+    {
+      "name": "Dummy with relations",
+      "relatedDummy": "http://example.com/related_dummies/1",
+      "relatedDummies": [
+        "/related_dummies/1",
+        "/related_dummies/2"
+      ],
+      "name_converted": null
+    }
+    """
+    Then the response status code should be 201
+    And the response should be in JSON
+    And the header "Content-Type" should be equal to "application/ld+json; charset=utf-8"
+    And the JSON should be equal to:
+    """
+    {
+      "@context": "/contexts/Dummy",
+      "@id": "/dummies/1",
+      "@type": "Dummy",
+      "description": null,
+      "dummy": null,
+      "dummyBoolean": null,
+      "dummyDate": null,
+      "dummyFloat": null,
+      "dummyPrice": null,
+      "relatedDummy": "/related_dummies/1",
+      "relatedDummies": [
+        "/related_dummies/1",
+        "/related_dummies/2"
+      ],
+      "jsonData": [],
+      "name_converted": null,
+      "id": 1,
+      "name": "Dummy with relations",
+      "alias": null
+    }
+    """
+
+  Scenario: Get the embedded relation collection
+    When I send a "GET" request to "/dummies/1/related_dummies"
+    And the response status code should be 200
+    And the response should be in JSON
+    And the header "Content-Type" should be equal to "application/ld+json; charset=utf-8"
+    And the JSON should be equal to:
+    """
+    {
+      "@context": "/contexts/RelatedDummy",
+      "@id": "/related_dummies",
```

Comment:
Hmm not sure because this is still a `related_dummy`. Anyway I really don't know how I can implement this because `getIriFromItem` isn't aware that we have requested this resource from a parent resource.